core/src/i18n/en.yaml

```diff
@@ -901,6 +943,11 @@ jobs:
   title: Jobs
 kubeconfig-id:
   error: "Couldn't load kubeconfig ID; configuration not changed (Error: ${{error}})."
+legal:
```

Comment:
Legal sections are alright. 

But I won't approve unless we know for sure that the translation workflow can accommodate split sentences

Reply:
sentences were combined again :)